packages/fluentui/accessibility/src/behaviors/Tree/treeAsListBehavior.ts

```diff
@@ -10,7 +10,7 @@ import { treeItemAsListItemBehavior } from './treeItemAsListItemBehavior';
  */
 export const treeAsListBehavior: Accessibility<TreeBehaviorProps> = props => {
   const behavior = treeBehavior(props);
-  return _.merge(behavior, {
+  const definition = _.merge(behavior, {
```

Comment:
Let's use spread here as well (see https://github.com/microsoft/fluentui/pull/15147/files#r504468796)

Reply:
Resolved by this [commit](https://github.com/microsoft/fluentui/pull/15147/commits/50290696e06d38de8212e6e56c1823df8fa9aa65)